ui/src/app/projects/projects.component.ts

```diff
@@ -95,11 +95,13 @@ export class ProjectsComponent implements OnInit {
 
   navigateJobs() {
     if (environment.dashboardEnabled) {
-      const dashboardExtras = {queryParams: {projectId: this.projectsControl.value}};
-      this.router.navigate(['dashboard'], dashboardExtras);
+      this.router.navigate(['dashboard'], {queryParams:
```

Comment:
Line wrapping looks messy here

Reply:
fixed.

Reply:
I don't see a change here. Here are reasonable styles in my opinion:

```
this.router.navigate(['dashboard'], {
  queryParams: {projectId: this.projectsControl.value}
});
```

or 

```
this.router.navigate(
    ['dashboard'], {queryParams: {projectId: this.projectsControl.value}});
```

Or just put it all on one line, if it fits

Reply:
Sorry, I got it wrong. Fixed.